Source/Meadow.ProjectLab/ProjectLabHardwareV1.cs

```diff
@@ -46,6 +46,10 @@ public ProjectLabHardwareV1(IF7FeatherMeadowDevice device, ISpiBus spiBus, II2cB
                         width: 240, height: 240,
                         colorMode: ColorType.Format16bppRgb565);
 
+            Display.SetRotation(TftSpiBase.Rotation.Rotate_270);

```

Comment:
is this call exposes anywhere besides the `base` class? feels weird calling a base method like that.

Reply:
It's defined on the ST7789 driver itself, it controls the way the hardware receives the display data

Reply:
thx! fixed .... although we might break the API again tomorrow ....